src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -41,10 +44,40 @@ public PreferencesView(DynamoViewModel dynamoViewModel)
             {
                 viewModel = viewModelTemp;
             }
-
+            
+            //Find all the Expanders in the LogicalTree (XAML)
```

Comment:
hmm Do you intend a call of `FindLogicalChildren` here?

Reply:
@QilongTang I forgot to delete that method because is not used any more, then I deleted the comment and the method
Thanks